src/analysis/LBOutputScreen.cpp

```diff
@@ -1,203 +1,186 @@
 /*
-  Copyright (C) 2016
-      Max Planck Institute for Polymer Research & JGU Mainz
-  Copyright (C) 2012,2013
-      Max Planck Institute for Polymer Research
-  Copyright (C) 2008,2009,2010,2011
-      Max-Planck-Institute for Polymer Research & Fraunhofer SCAI
+ Copyright (C) 2012-2016 Max Planck Institute for Polymer Research
```

Comment:
Why erasing the whole (c) here? for one line? look at commit #91 for solving it!